components/display/index.rst

```diff
@@ -134,7 +134,8 @@ specific sizes, with ESPHome you have the option to use **any** TrueType (``.ttf
 at **any** size! Granted the reason for it is actually not having to worry about the licensing of font files :)
 
 To use fonts you first have to define a font object in your ESPHome configuration file. Just grab
-a ``.ttf`` file from somewhere on the Internet and create a ``font:`` section in your configuration:
+a ``.ttf`` file from somewhere on the internet and upload it to the ``config/esphome`` folder.
```

Comment:
The `config/esphome` folder is very specific for Home Assistant installation of a certain type (e.g., Home Assistant OS and Supervised), and thus does not apply to all other users.

Can we change the wording to avoid that?